paasta_tools/autoscale_cluster.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import logging
+
+from paasta_tools.autoscaling_lib import autoscale_local_cluster
+
+
+log = logging.getLogger('__main__')
```

Comment:
Hm, I don't think this is the 'right' way to do logging.

I think you should have:
```
log = logging.getLogger(__name__)
```

Then in ``autoscaling_lib.py`` have:

```
log = logging.getLogger(__name__).addHandler(logging.NullHandler())
```

That way, the logging in ``autoscaling_lib.py`` will respect the logging configuration of whatever is using the library (the use of the NullHandler means that if logging hasn't been setup, then it won't spew unwanted output, and won't complain about missing handlers), and you'll see the correct source of the log line in the output